EventListener/RedirectListener.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the FOSUserBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\UserBundle\EventListener;
+
+use FOS\UserBundle\Event\FormEvent;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\Routing\RouterInterface;
+
+/**
+ * Redirect to a custom URL in response to FormEvent dispatches.
+ */
+class RedirectListener
+{
+    /**
+     * @var RouterInterface
+     */
+    private $router;
+
+    /**
+     * @var string
+     */
+    private $route;
+
+    /**
+     * @param RouterInterface $router
+     * @param string $route
+     */
+    public function __construct(RouterInterface $router, $route)
```

Comment:
Ah that makes sense, thanks.

Should I update the PR or will this not be merged? It seems generic and common enough that most FOSUB users will need something like it. I keep on adding it to projects, anyway. Of course I could handle that differently if it won't be merged.